code/datums/martial/holy_crunch.dm

```diff
@@ -0,0 +1,47 @@
+/datum/martial_art/hand_of_god
+	name = "Hand of God"
+	deflection_chance = 0
+	no_guns = TRUE
+	allow_temp_override = FALSE
+
+/datum/martial_art/hand_of_god/harm_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	A.do_attack_animation(D, ATTACK_EFFECT_PUNCH)
+	var/attack_type = pick("brute", "burn", "toxin", "knockdown")
+	if(attack_type == "brute")
+		var/atk_verb = pick("power word: fracture", "power word: break", "power word: crack", "power word: dust", "power word: fissure")
+		D.visible_message("<span class='danger'>[A] uses [atk_verb] on [D]!</span>", \
+					  "<span class='userdanger'>[A] uses [atk_verb] on you!</span>")
+		D.apply_damage(10, BRUTE)
+		playsound(get_turf(D), 'sound/magic/repulse.ogg', 25, 1, -1)
+		add_logs(A, D, " used [atk_verb] (Hand of God) on")
+		return 1
+	if(attack_type == "burn")
+		var/atk_verb = pick("power word: scorch", "power word: sear", "power word: burn", "power word: fireball", "power word: combust")
+		D.visible_message("<span class='danger'>[A] uses [atk_verb] on [D]!</span>", \
+					  "<span class='userdanger'>[A] uses [atk_verb] on you!</span>")
+		D.apply_damage(10, BURN)
+		playsound(get_turf(D), 'sound/magic/teleport_app.ogg', 25, 1, -1)
+		add_logs(A, D, " used [atk_verb] (Hand of God) on")
+		return 1
+	if(attack_type == "toxin")
+		var/atk_verb = pick("power word: impair", "power word: afflict", "power word: debilitate", "power word: hurt", "power word: diminish", )
+		D.visible_message("<span class='danger'>[A] uses [atk_verb] on [D]!</span>", \
+					  "<span class='userdanger'>[A] uses [atk_verb] on you!</span>")
+		D.apply_damage(10, TOX)
+		playsound(get_turf(D), 'sound/effects/bamf.ogg', 25, 1, -1)
+		add_logs(A, D, " used [atk_verb] (Hand of God) on")
+		return 1
+	if(attack_type == "knockdown")
+		var/atk_verb = pick("power word: stasis", "power word: daze", "power word: tranquilise", "power word: benumb", "power word: ban", )
```

Comment:
Power word: Ban? That's some OOC in IC if I've seen it before.

Reply:
What, are you saying spessmen never played a game involving the galaxy wide internet to not know the word 'ban'?

Reply:
ban
officially or legally prohibit.

or even better

archaic
a curse.

Reply:
Those arguments are really lukewarm, but I guess I won't be turning any heads anyways.